plugin/mutate/source/dextool/plugin/mutate/backend/report/html/page_long_term_view.d

```diff
@@ -49,9 +49,9 @@ void toHtml(const MutantSample mut_sample, Element root) {
     root.addChild("h2", "High Interest Mutants");
 
     if (mut_sample.hardestToKill.length != 0) {
-        root.addChild("h3", "Longest Surviving Mutant");
+        root.addChild("h3", "Longest surviving Mutants");
         root.addChild("p",
-                "These mutants has survived countless test runs. Slay one or more of them to be the hero of the team.");
```

Comment:
You are not the first one that disliked my try at being funny ;)

Reply:
If I get a vote, I would say keep the comedy! :)